src/app/services/logger/exui-logger/exui-logger.component.ts

```diff
@@ -0,0 +1,66 @@
+import {Component, OnInit, OnDestroy} from '@angular/core';
+import { NGXLogger } from 'ngx-logger';
+import { LoggerMessage } from 'src/app/services/logger/reducers/logger.reducer';
+import { Store, select } from '@ngrx/store';
+import { getDebugMessagesState, getInfoMessagesState, getWarningMessagesState, getErrorMessagesState,
+  getFatalMessagesState, getTraceMessagesState } from 'src/app/services/logger/selectors/logger.selector';
+
+@Component({
+  selector: 'exui-logger',
+  template: `
+    <div>
+    <div>
+  `
+})
+export class ExuiLoggerComponent implements OnInit, OnDestroy {
+
+  debugSubscribtion: any;
+  infoSubscribtion: any;
+  warningSubscribtion: any;
+  errorSubscribtion: any;
+  fatalSubscribtion: any;
+  traceSubscribtion: any;
+
+  constructor(private ngxLogger: NGXLogger, private store: Store<LoggerMessage>) { }
+  ngOnInit(): void {
```

Comment:
here needs to come code from app.component ngOnInit and the component name needs to be dynamic